### DISCUSSION TITLE
removeLabel cannot find label which exist

### DISCUSSION THREAD

**Discussion by rummunkey** [octokit/octokit.js#2313] [Q&A]
I can listLabelsOnIssue and confirm the existence of a label but for some reason when I give that specific label to removeLabel it returns the error that it cannot find the specific label even though I can see it being printed out of console log with the same function.

**Comment by gr2m** [comment]
Did you by chance rename the repository and set `repo` to the previous name? I think there is currently a bug in GitHub's APIs that redirects some of the endpoints to new repository name, but not all of them yet.

**Reply by rummunkey** [reply]
Yes the repo previously went by a different name before my time.

**Reply by gr2m** [reply]
Make sure in your `octokit.rest.repos.listLabelsOnIssue({ owner, repo, issue_numer })` call that you set `repo` to the new repository name? Does that resolve your problem?

**Reply by rummunkey** [reply]
Listlabels works perfect it's removeLabel that return that the label is not found.

**Reply by rummunkey** [reply]
Using the same label I found from the list and same repo as the list function.

**Reply by gr2m** [reply]
yeah sorry I meant in the method call that removes the label.

Can you share the full error you are getting? The error should include all request options, that will make it easier to debug
